quickstarts/deeper-network/config.yml

```diff
@@ -0,0 +1,40 @@
+name: deeper-network
+
+title: Deeper Network
+
+description: |+
+  ## Deeper Connect Monitoring Quickstart
+
+  Get deep visibility into the health, performance, connections, logs and balance of your Deeper Connect device.
+
+  Alert and identify potential issues, or simply use for remote monitoring.
+ 
+summary: |+
+  Monitor and alert on the health of your Deeper Connect devices to ensure ongoing reliability and performance. Requires deeper-cli setup.
+
+
+level: Community
+
+icon: icon.png
+logo: logo.png
+website: https://deeper.network/
+
+authors:
+  - New Relic
+  - Kav. Pather
+
+documentation:
+  - name: Deeper Network
+    description: |
+      The Decentralized Gateway and Infrastructure for Web3.0. Deeper Network combines blockchain, network security, and sharing economy to create a global peer to peer network that empowers the real users of the internet and paves the way for the next generation of the web.
+    url: https://deeper.network/
```

Comment:
Is there a specific docs page or GH readme that explains the steps for setting up Deeper Connect with New Relic?

Reply:
I intentionally did not add something specific right now, was just trying to coordinate when & where I make the guide & CLI utility public, if potentially under their website/github. 
But for better coverage right now I've added another documentation link and install plan, I will make it public when this PR is merged and/or the CLI utility made public. 

Hoping this should be sufficient for now from our end?